client/src/pages/admin/users/UserModal.tsx

```diff
@@ -101,6 +110,26 @@ export default function UserModal({ selectedUserID, onClose }: UserModalProps) {
     });
   };
 
+  const handleDeleteUserClicked = () => {
+      let fName = getUserResult.data.user.firstName;
+      let lName = getUserResult.data.user.lastName;
+      let privilege = getUserResult.data.user.privilege;
```

Comment:
Let's use `const` instead of `let` whenever we're not reassigning the variable. Same goes for `result` below.